### DISCUSSION TITLE
Write a new file using pdfplumber?

### DISCUSSION THREAD

**Discussion by awa5114** [jsvine/pdfplumber#440] [Q&A]
I was successfully able to extract a crop of a page using `pdfplumber`'s `page.within_bbox` method.

I would like to ask whether it is possible to write a new file which contains only this page along with its cropped content? Thanks for your consideration.

**Comment by samkit-jain** [comment]
Hi @awa5114 Appreciate your interest in the library. No, it is not possible to create/modify a PDF using `pdfplumber`. It is focused on extracting information from PDFs, rather than creating/modifying them. You may look at tools like [pdfrw](https://github.com/pmaupin/pdfrw) for write operations on a PDF. You can export a page as an image with a few drawing operations available however.

**Reply by johnsnow09** [reply]
Hi @samkit-jain I have tried saving cropped pages of `pdfplumber` using `pdfrw` and it is giving error. Have also posted this on Stackover flow at [https://stackoverflow.com/questions/79222706/how-to-save-pdf-after-cropping-from-each-page-of-pdf-using-pdfplumber](url). 

Could you please have a look and suggest how to save pdf after cropping portions of each page of pdf.

```
output_pdf =  PdfWriter() 

with pdfplumber.open(path) as pdf:
    pages = pdf.pages

    for p in pages:
        print(p)
        bbox_vals = p.find_tables()[0].bbox
        y0_top_table = bbox_vals[3]

        print(y0_top_table)
        print(round(y0_top_table))
        cropped_pdf = p.crop((0, y0_top_table, 590, 840))
        print(type(cropped_pdf))

        output_pdf.addpage(cropped_pdf)
```  

Thanks !!
